src/adagrams.js

```diff
@@ -1,8 +1,94 @@
-const Adagrams = {
-  drawLetters() {
+class Adagrams {
+  static drawLetters() {
+    const adagramsCount = {
+       'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+    };
+
+    const letterPool = [];
+
+    for (let key in adagramsCount) {
+      for (let i = 0; i < adagramsCount[key]; i++) {
```

Comment:
I would really advise against using these types of `for` loops and instead using the `forEach` or for-in etc.